tests/utils.py

```diff
@@ -173,13 +207,134 @@ def wrapper(*args, **kwargs):
+        if camera_type is None:
+            raise ValueError("The 'camera_type' must be an argument of the test function.")
+        if mock is None:
+            raise ValueError("The 'mock' variable must be an argument of the test function.")
+
+        if not mock and not request.getfixturevalue("is_camera_available"):
+            pytest.skip(f"A {camera_type} camera is not available.")
+
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+def require_motor(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        # Access the pytest request context to get the is_motor_available fixture
+        request = kwargs.get("request")
+        motor_type = kwargs.get("motor_type")
+        mock = kwargs.get("mock")
+
+        if request is None:
+            raise ValueError("The 'request' fixture must be an argument of the test function.")
+        if motor_type is None:
+            raise ValueError("The 'motor_type' must be an argument of the test function.")
+        if mock is None:
+            raise ValueError("The 'mock' variable must be an argument of the test function.")
+
+        if not mock and not request.getfixturevalue("is_motor_available"):
+            pytest.skip(f"A {motor_type} motor is not available.")
+
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+def make_robot(robot_type: str, overrides: list[str] | None = None, mock=False) -> Robot:
+    if mock:
+        overrides = [] if overrides is None else copy(overrides)
+
+        if robot_type == "koch":
+            overrides.append("+leader_arms.main.mock=true")
+            overrides.append("+follower_arms.main.mock=true")
+            if "~cameras" not in overrides:
+                overrides.append("+cameras.laptop.mock=true")
+                overrides.append("+cameras.phone.mock=true")
+
+        elif robot_type == "koch_bimanual":
+            overrides.append("+leader_arms.left.mock=true")
+            overrides.append("+leader_arms.right.mock=true")
+            overrides.append("+follower_arms.left.mock=true")
+            overrides.append("+follower_arms.right.mock=true")
+            if "~cameras" not in overrides:
+                overrides.append("+cameras.laptop.mock=true")
+                overrides.append("+cameras.phone.mock=true")
+
+        elif robot_type == "aloha":
+            overrides.append("+leader_arms.left.mock=true")
+            overrides.append("+leader_arms.right.mock=true")
+            overrides.append("+follower_arms.left.mock=true")
+            overrides.append("+follower_arms.right.mock=true")
+            if "~cameras" not in overrides:
+                overrides.append("+cameras.cam_high.mock=true")
+                overrides.append("+cameras.cam_low.mock=true")
+                overrides.append("+cameras.cam_left_wrist.mock=true")
+                overrides.append("+cameras.cam_right_wrist.mock=true")
```

Comment:
Is there a better way?

Reply:
It would require to load the hydra yaml and dynamically look for cameras, etc.
As of now, I prefer something simple and explicit which is prone to break.

I added:
```
# Explicitely add mock argument to the cameras and set it to true
# TODO(rcadene, aliberts): redesign when we drop hydra
```